source/docs/guides/apple_pay/index.md

```diff
@@ -161,6 +176,25 @@ func paymentAuthorizationViewController(_ controller: PKPaymentAuthorizationView
 }
 ```
 
+>It's very important to notice that the ***payment_token is Base64 encoded*** including the paymentData.
+
+>Below it's shown an example of the paymentData object that should be submitted.
```

Comment:
Change to:
"Below is an example of the paymentData object that should be submitted."

Reply:
Done!